securesystemslib/signer.py

```diff
@@ -0,0 +1,127 @@
+"""
+<Program Name>
+  signer.py
+
+<Author>
+  Martin Vrachev <mvrachev@vmware.com>
+
+<Started>
+  Januart 27, 2021.
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  The goal of this module is to provide signing interface supporting multiple
+  signing implementations.
+"""
+
+import abc
+import securesystemslib.keys as sslib_keys
+
+
+
+class Signature:
+  """
+  <Purpose>
+    Storage class containing information about a signature and the keyid
+    uniquely identifying the key used to generate the signature.
+
+  <Attributes>
+    keyid: HEX string used as a unique identifier of the key.
+    signature: HEX string representing the signature.
+  """
+  def __init__(self, keyid, sig):
+    self.keyid = keyid
+    self.signature = sig
+
+
+  def to_dict(self):
```

Comment:
If you provide a `to_dict`, I suggest you also provide a `from_dict` classmethod.